From e96cffa494b4a818372acf1ff5a30329e1496238 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Tue, 26 Feb 2019 16:58:28 +0100
Subject: [PATCH] Use correct i18n include

From the Internationalization section of the GLib API docs:
In order to use these macros in an application, you must include
<glib/gi18n.h>. For use in a library, you must include <glib/gi18n-lib.h>
after defining the GETTEXT_PACKAGE macro suitably for your library

Applied-Upstream: in upstream libhandy, g-c-c didn't pull the new copy
into its submodule at the time of writing
---
 subprojects/libhandy/src/hdy-action-row.c          | 3 ++-
 subprojects/libhandy/src/hdy-arrows.c              | 3 ++-
 subprojects/libhandy/src/hdy-column.c              | 3 ++-
 subprojects/libhandy/src/hdy-combo-row.c           | 3 ++-
 subprojects/libhandy/src/hdy-dialer-button.c       | 3 ++-
 subprojects/libhandy/src/hdy-dialer-cycle-button.c | 3 ++-
 subprojects/libhandy/src/hdy-dialer.c              | 3 ++-
 subprojects/libhandy/src/hdy-dialog.c              | 3 ++-
 subprojects/libhandy/src/hdy-expander-row.c        | 3 ++-
 subprojects/libhandy/src/hdy-fold.c                | 1 +
 subprojects/libhandy/src/hdy-header-group.c        | 3 ++-
 subprojects/libhandy/src/hdy-leaflet.c             | 3 ++-
 subprojects/libhandy/src/hdy-list-box.c            | 3 ++-
 subprojects/libhandy/src/hdy-main.c                | 2 +-
 subprojects/libhandy/src/hdy-search-bar.c          | 3 ++-
 subprojects/libhandy/src/hdy-title-bar.c           | 3 ++-
 subprojects/libhandy/src/hdy-value-object.c        | 3 ++-
 17 files changed, 32 insertions(+), 16 deletions(-)

diff --git a/subprojects/libhandy/src/hdy-action-row.c b/src/hdy-action-row.c
index fc12f12..4467cc7 100644
--- a/subprojects/libhandy/src/hdy-action-row.c
+++ b/subprojects/libhandy/src/hdy-action-row.c
@@ -4,9 +4,10 @@
  * SPDX-License-Identifier: LGPL-2.1+
  */
 
+#include "config.h"
 #include "hdy-action-row.h"
 
-#include <glib/gi18n.h>
+#include <glib/gi18n-lib.h>
 
 /**
  * SECTION:hdy-action-row
diff --git a/subprojects/libhandy/src/hdy-arrows.c b/src/hdy-arrows.c
index d6e512c..26bce3d 100644
--- a/subprojects/libhandy/src/hdy-arrows.c
+++ b/subprojects/libhandy/src/hdy-arrows.c
@@ -4,7 +4,8 @@
  * SPDX-License-Identifier: LGPL-2.1+
  */
 
-#include <glib/gi18n.h>
+#include "config.h"
+#include <glib/gi18n-lib.h>
 
 #include "hdy-arrows.h"
 #include "hdy-enums.h"
diff --git a/subprojects/libhandy/src/hdy-column.c b/src/hdy-column.c
index 67c69b2..387c80f 100644
--- a/subprojects/libhandy/src/hdy-column.c
+++ b/subprojects/libhandy/src/hdy-column.c
@@ -4,9 +4,10 @@
  * SPDX-License-Identifier: LGPL-2.1+
  */
 
+#include "config.h"
 #include "hdy-column.h"
 
-#include <glib/gi18n.h>
+#include <glib/gi18n-lib.h>
 #include <math.h>
 
 /**
diff --git a/subprojects/libhandy/src/hdy-combo-row.c b/src/hdy-combo-row.c
index e96c7c4..a06fddf 100644
--- a/subprojects/libhandy/src/hdy-combo-row.c
+++ b/subprojects/libhandy/src/hdy-combo-row.c
@@ -4,9 +4,10 @@
  * SPDX-License-Identifier: LGPL-2.1+
  */
 
+#include "config.h"
 #include "hdy-combo-row.h"
 
-#include <glib/gi18n.h>
+#include <glib/gi18n-lib.h>
 #include "hdy-list-box.h"
 
 /**
diff --git a/subprojects/libhandy/src/hdy-dialer-button.c b/src/hdy-dialer-button.c
index 488a55e..e9ef1fd 100644
--- a/subprojects/libhandy/src/hdy-dialer-button.c
+++ b/subprojects/libhandy/src/hdy-dialer-button.c
@@ -4,7 +4,8 @@
  * SPDX-License-Identifier: LGPL-2.1+
  */
 
-#include <glib/gi18n.h>
+#include "config.h"
+#include <glib/gi18n-lib.h>
 
 #include "hdy-dialer-button.h"
 
diff --git a/subprojects/libhandy/src/hdy-dialer-cycle-button.c b/src/hdy-dialer-cycle-button.c
index ed94e56..899ba12 100644
--- a/subprojects/libhandy/src/hdy-dialer-cycle-button.c
+++ b/subprojects/libhandy/src/hdy-dialer-cycle-button.c
@@ -4,7 +4,8 @@
  * SPDX-License-Identifier: LGPL-2.1+
  */
 
-#include <glib/gi18n.h>
+#include "config.h"
+#include <glib/gi18n-lib.h>
 
 #include "hdy-dialer-cycle-button.h"
 
diff --git a/subprojects/libhandy/src/hdy-dialer.c b/src/hdy-dialer.c
index 78aac94..66fd6ca 100644
--- a/subprojects/libhandy/src/hdy-dialer.c
+++ b/subprojects/libhandy/src/hdy-dialer.c
@@ -4,7 +4,8 @@
  * SPDX-License-Identifier: LGPL-2.1+
  */
 
-#include <glib/gi18n.h>
+#include "config.h"
+#include <glib/gi18n-lib.h>
 
 #include "hdy-dialer.h"
 #include "hdy-dialer-button.h"
diff --git a/subprojects/libhandy/src/hdy-dialog.c b/src/hdy-dialog.c
index 283c438..1557e1c 100644
--- a/subprojects/libhandy/src/hdy-dialog.c
+++ b/subprojects/libhandy/src/hdy-dialog.c
@@ -4,7 +4,8 @@
  * SPDX-License-Identifier: LGPL-2.1+
  */
 
-#include <glib/gi18n.h>
+#include "config.h"
+#include <glib/gi18n-lib.h>
 
 #include "hdy-dialog.h"
 
diff --git a/subprojects/libhandy/src/hdy-expander-row.c b/src/hdy-expander-row.c
index 2174c3e..a58249b 100644
--- a/subprojects/libhandy/src/hdy-expander-row.c
+++ b/subprojects/libhandy/src/hdy-expander-row.c
@@ -4,9 +4,10 @@
  * SPDX-License-Identifier: LGPL-2.1+
  */
 
+#include "config.h"
 #include "hdy-expander-row.h"
 
-#include <glib/gi18n.h>
+#include <glib/gi18n-lib.h>
 
 /**
  * SECTION:hdy-expander-row
diff --git a/subprojects/libhandy/src/hdy-fold.c b/src/hdy-fold.c
index e3a9a22..7029cd0 100644
--- a/subprojects/libhandy/src/hdy-fold.c
+++ b/subprojects/libhandy/src/hdy-fold.c
@@ -4,6 +4,7 @@
  * SPDX-License-Identifier: LGPL-2.1+
  */
 
+#include "config.h"
 #include "hdy-fold.h"
 
 /**
diff --git a/subprojects/libhandy/src/hdy-header-group.c b/src/hdy-header-group.c
index d6640b8..864e476 100644
--- a/subprojects/libhandy/src/hdy-header-group.c
+++ b/subprojects/libhandy/src/hdy-header-group.c
@@ -4,7 +4,8 @@
  * SPDX-License-Identifier: LGPL-2.1+
  */
 
-#include <glib/gi18n.h>
+#include "config.h"
+#include <glib/gi18n-lib.h>
 
 #include "hdy-header-group.h"
 
diff --git a/subprojects/libhandy/src/hdy-leaflet.c b/src/hdy-leaflet.c
index 3462a9c..c970aad 100644
--- a/subprojects/libhandy/src/hdy-leaflet.c
+++ b/subprojects/libhandy/src/hdy-leaflet.c
@@ -4,7 +4,8 @@
  * SPDX-License-Identifier: LGPL-2.1+
  */
 
-#include <glib/gi18n.h>
+#include "config.h"
+#include <glib/gi18n-lib.h>
 
 #include "gtkprogresstrackerprivate.h"
 #include "hdy-leaflet.h"
diff --git a/subprojects/libhandy/src/hdy-list-box.c b/src/hdy-list-box.c
index 8faac6c..1290116 100644
--- a/subprojects/libhandy/src/hdy-list-box.c
+++ b/subprojects/libhandy/src/hdy-list-box.c
@@ -4,9 +4,10 @@
  * SPDX-License-Identifier: LGPL-2.1+
  */
 
+#include "config.h"
 #include "hdy-list-box.h"
 
-#include <glib/gi18n.h>
+#include <glib/gi18n-lib.h>
 
 /**
  * SECTION:hdy-list-box
diff --git a/subprojects/libhandy/src/hdy-main.c b/src/hdy-main.c
index 004e1bf..08ef1e3 100644
--- a/subprojects/libhandy/src/hdy-main.c
+++ b/subprojects/libhandy/src/hdy-main.c
@@ -6,7 +6,7 @@
 #include "config.h"
 #include "hdy-main-private.h"
 #include <gio/gio.h>
-#include <glib/gi18n.h>
+#include <glib/gi18n-lib.h>
 
 static gint hdy_initialized = FALSE;
 
diff --git a/subprojects/libhandy/src/hdy-search-bar.c b/src/hdy-search-bar.c
index 58c9f88..1f18ca7 100644
--- a/subprojects/libhandy/src/hdy-search-bar.c
+++ b/subprojects/libhandy/src/hdy-search-bar.c
@@ -41,7 +41,8 @@
  * the fork.
  */
 
-#include <glib/gi18n.h>
+#include "config.h"
+#include <glib/gi18n-lib.h>
 
 #include "hdy-search-bar.h"
 
diff --git a/subprojects/libhandy/src/hdy-title-bar.c b/src/hdy-title-bar.c
index ed8f1cd..795c7c5 100644
--- a/subprojects/libhandy/src/hdy-title-bar.c
+++ b/subprojects/libhandy/src/hdy-title-bar.c
@@ -4,9 +4,10 @@
  * SPDX-License-Identifier: LGPL-2.1+
  */
 
+#include "config.h"
 #include "hdy-title-bar.h"
 
-#include <glib/gi18n.h>
+#include <glib/gi18n-lib.h>
 
 /**
  * SECTION:hdy-title-bar
diff --git a/subprojects/libhandy/src/hdy-value-object.c b/src/hdy-value-object.c
index 3adbb2b..485d4b3 100644
--- a/subprojects/libhandy/src/hdy-value-object.c
+++ b/subprojects/libhandy/src/hdy-value-object.c
@@ -4,7 +4,8 @@
  * SPDX-License-Identifier: LGPL-2.1+
  */
 
-#include <glib/gi18n.h>
+#include "config.h"
+#include <glib/gi18n-lib.h>
 #include <gobject/gvaluecollector.h>
 #include "hdy-value-object.h"
 
-- 
2.20.1

